build.ps1

```diff
@@ -1322,6 +1322,16 @@ function Install-HostToolchain() {
   Copy-Item -Force $BinaryCache\7\bin\swift-driver.exe $ToolchainInstallRoot\usr\bin\swiftc.exe
 }
 
+function Build-Inspect() {
+  $OutDir = Join-Path -Path $HostArch.BinaryRoot -ChildPath swift-inspect
```

Comment:
Fancier than everywhere else's `"$($HostArch.BinaryRoot)\swift-inspect"`!

Reply:
We should clean that up eventually and use `Join-Path`.